media/js/mozorg/contact-spaces.js

```diff
@@ -115,12 +115,16 @@
             //set the current tab navigation item
             $('ul.category-tabs li[data-id="' + tab + '"]').addClass('current');
 
-            // set the current list menu navigation item
-            $('.nav-category li[data-id="' + id + '"]').addClass('current');
+            if (tab === 'spaces') {
+                // set the current list menu navigation item for the active space
+                $('.nav-category li[data-id="' + id + '"]').addClass('current');
```

Comment:
Ah, I see it's repeated below. Would be easier/more DRY to just have the line once, not wrapped in a conditional though, yeah?